std_msgs/Header header
TrackedObject[] tracked_objects_vector
